tests/cypress/integration/issue_1599_ch_user_registration.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Issue 1599 (Chinese alphabet).', () => {
+    before(() => {
+        cy.visit('auth/register')
+        cy.url().should('include', '/auth/register')
+    })
+
+    describe('User registration using the Chinese alphabet.', () => {
+        it('Filling in the plaseholder "First name"', () => {
+          cy.get('[placeholder="First name"]').type('测试者')
+          cy.contains('Please specify a first name').should('not.exist')
```

Comment:
I would suggest to check this test in following way: 

``cy.get('[placeholder="First name"]').should('not.have.class', 'has-error')``

Because we can just change these messages and it would be better if tests won't be broken in this case.

Reply:
Done. Replaced.